applications/NXxas.nxdl.xml

```diff
@@ -109,6 +109,7 @@
                 <doc>preset value for time or monitor</doc>
             </field>
             <field name="data" type="NX_INT">
+		<doc>This field could be a link to ``/entry/instrument/incoming_beam/data``</doc>
```

Comment:
What is `/entry/instrument/incoming_beam/data`? Do you mean to say that this field can be a link to the data field of an NXmonitor which resides under NXinstrument? That's weirdly specific. Application definitions are full of those types of fields.

Reply:
Yes, this is suggesting a link to the `incoming_beam` detector. It is weirdly specific, but that is how the author of the application definition seems to have intended it.